pkgs/tools/graphics/wkhtmltopdf-bin/default.nix

```diff
@@ -57,16 +58,24 @@ let
 
       (lib.getLib fontconfig)
       (lib.getLib gcc-unwrapped)
-      (lib.getLib libjpeg8)
+      (lib.getLib libjpeg)
       (lib.getLib libpng)
     ];
 
-    unpackPhase = "tar -xf $src";
+    unpackPhase = ''
+      runHook preUnpack
+
+      mkdir pkg
+      dpkg-deb -x $src pkg
```

Comment:
Does fetchzip decompress it? idk

Reply:
Technally yes, there is one package doing it, pkgs/by-name/te/tetrio-desktop/package.nix. All of the others are using fetchurl.

I prefer using fetchurl as the hash is on the content and can be easily calculated separately.